modules/spring-config/pom.xml

```diff
@@ -84,6 +84,12 @@
             <version>${springCglibVersion}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.ws.xmlschema</groupId>
+            <artifactId>xmlschema-core</artifactId>
+            <version>2.2.1</version>
```

Comment:
that lib is there because I'm generating the XSD from a `ModuleExtension` object (which is a representation of a module-some-connector.xml file).
In a future, there will be an `ExtensionModel` rather than a `ModuleExtension`, and there's a current support for generating XSD for `ExtensionModel`.

@pablolagreca: Do you believe it's better to move that version upto the mule/pom.xml file rather than leave it in the current spring-config module?